examples/plugins/c-json-filetype/filetype.c

```diff
@@ -140,12 +125,12 @@ static void FiletypeDeinit(void *data)
  * of "eve.<thread_id>.json". This plugin may want to do similar, or open
  * multiple connections to whatever the final logging location might be.
  *
- * In the case of non-threaded EVE logging this function is NOT called by
- * Suricata, but instead this plugin chooses to use this method to create a
- * default (single) thread context.
+ * In the case of non-threaded EVE logging this function is called
+ * once with a thread_id of 0.
  */
 static int FiletypeThreadInit(void *ctx, ThreadId thread_id, void **thread_data)
 {
+    SCLogNotice("thread_id=%d", thread_id);
```

Comment:
are these notice statements supposed to go as is?

Reply:
Yes --- this code is for demonstrative purposes only for an example output plugin.